<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeDbSelectDataPools</class>
 <widget class="QWidget" name="C_SdNdeDbSelectDataPools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_SdNdeDbSelectDataPools</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>27</number>
   </property>
   <property name="topMargin">
    <number>27</number>
   </property>
   <property name="rightMargin">
    <number>27</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgeGbx" name="pc_GroupBoxGeneral_5">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>13</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="stw::opensyde_gui_elements::C_OgeLabHeadingGroupBold" name="pc_LabelHeadingPreview">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="stw::opensyde_gui_elements::C_OgeFraSeparator" name="pc_Frame_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="stw::opensyde_gui_elements::C_OgeLeProperties" name="pc_LineEditSearch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelSelection">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="stw::opensyde_gui::C_TblTreDataElementView" name="pc_TreeView"/>
      </item>
      <item>
       <widget class="stw::opensyde_gui_elements::C_OgeGbxTransparent" name="pc_GroupBoxSearchNoElementsFound">
        <property name="title">
         <string>Search - No element case:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>259</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelSearchNoElementsFound">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>259</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="stw::opensyde_gui_elements::C_OgeGbxTransparent" name="pc_GroupBoxInitialDataElementNoElements">
        <property name="title">
         <string>Initial - Data element - No element case:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>259</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelInitialDataElementsNoElements">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>259</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="pc_VerticalSpacerButtonsObove">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgePubCancel" name="pc_PushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgePubDialog" name="pc_PushButtonOk">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabHeadingGroupBold</class>
   <extends>QLabel</extends>
   <header>C_OgeLabHeadingGroupBold.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeFraSeparator</class>
   <extends>QFrame</extends>
   <header>C_OgeFraSeparator.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLeProperties</class>
   <extends>QLineEdit</extends>
   <header>C_OgeLeProperties.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeGbxTransparent</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxTransparent.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgePubDialog</class>
   <extends>QPushButton</extends>
   <header>C_OgePubDialog.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeGbx</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbx.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgePubCancel</class>
   <extends>QPushButton</extends>
   <header>C_OgePubCancel.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_TblTreDataElementView</class>
   <extends>QTreeView</extends>
   <header>C_TblTreDataElementView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
